SteamBundleSitesExtension.js

```diff
@@ -744,12 +763,28 @@ const bundleSitesBox = () => {
             cursor: pointer;
             transition: all 0.5s;
         }
+        .SBSE_container a {
+            display: inline-block;
+            text-align: center;
+            width: 120px;
+            position: relative;
+            margin-right: 10px;
+            line-height: 28px;
+            box-sizing: border-box;
+            outline: none;
+            cursor: pointer;
+            transition: all 0.5s;
+        }
```

Comment:
Better merge `.SBSE_container a` with `.SBSE_container a` and remove all styles except display & text-align, these two basically have the same styles

Reply:
Fix in next commit~ XD

Reply:
Is it not allowed to edit pull request?
I think it's better to fix all changes and then merge

Reply:
if you think this pull request is not perfect BUT it nice
you can receive this pull request 
then fix the code style or glitch by yourself

if you add me to Collaborators, so that I can pull my code directly and you can edit my code too